packages/clerk-js/playground.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
```

Comment:
This playground file is something I built alongside the feature to help me to develop it. This requires little to no setup - you just add your API key, open the file, and its off to the races. I also feel like having a pure js testbed for things is helpful in general - I turned up a number of bugs that I bet are not present in higher abstractions like react & nextjs libs.

Reply:
This is great, I wonder if we should move this inside the top-level `playground` dir instead